core/templates/pages/community-dashboard-page/community-dashboard-page.directive.html

```diff
@@ -223,4 +224,14 @@
     color: #fff;
     padding: 1px 5px;
   }
+  .oppia-community-message-bar {
+    background-color: #00376d;
```

Comment:
How was this color chosen? I'm wondering if there is a standard banner color we should use across Oppia.

Reply:
Yeah, this is similar to the color of the navbar of the admin page. I'm fine with changing the style or color, let me know if you have any suggestion?
![image](https://user-images.githubusercontent.com/16653571/80410532-58185000-88e8-11ea-8bc7-ae4a3b09bad9.png)

Reply:
Wouldn't you want a different color than that of the admin nav bar for a notification message? Does Oppia have some kind of design language for things like this? Common colors, fonts, etc.

Reply:
We already have the promo bar. I suggest doing something similar.